tasks/README.md

```diff
@@ -329,6 +329,9 @@ Name | Summary | Category
 `task382_hybridqa_answer_generation` | Given a question about part-of-speech tag of a word in the question, answer the question. | Answer Generation
 `task386_semeval_2018_task3_irony_detection` | Given a tweet judge whether it contains irony or not. | Classification
 `task387_semeval_2018_task3_irony_classification` | Given a tweet Classify the kind of irony it has. | Classification
+`task388_torque_event_detection` | Given a passage, identify an event from it | Event Detection
```

Comment:
I'm not sure if "event detection", "question generation", or "answer generation" fit well into your taxonomy here. I guess no because you have such a big repo of different tasks. The three tasks of TORQUE are actually:
- event detection --> token classification
- question generation --> text generation
- answer generation --> text span selection
I hope that helps.

Reply:
Though your comments are valuable, I am not sure if I should be defining new categories. 
@danyaljj @swarooprm let me know if you would like to incorporate above changes.

Reply:
Thanks @qiangning for your comments. I agree that, we should create token classification and textspan selection as new categories because event detection is too specific and answer generation is too general.
However, I think it may be a good idea to keep question generation as it is because we have multiple text generation tasks which we have named differently e.g. long text generation, sentence generation, question generation etc.
@shailaja183, when you fix this PR, also create an issue to rename some of the existing answer generation tasks to text span selection and classification tasks to token classification wherever appilcable.